amp-script-tutorial/README.md

```diff
@@ -0,0 +1,29 @@
+# amp-script-tutorial
+
+## About
+
+This repository contains the code you'll use to do the [&lt;amp-script&gt; tutorial](https://amp.dev/documentation/guides-and-tutorials/develop/custom-javascript-tutorial/)!
+
+This tutorial presents a form containing a password input. Any password entered in this input must meet certain criteria, and so the form won't allow the user to submit until it does. In the tutorial, you begin with HTML that enforces the rules using &lt;amp-form&gt;'s `pattern` attribute. You then transform that into a more user-friendly experience which uses &lt;amp-script&gt;.
+
+This repo contains two subdirectories:
+* `starter_code`: the code you start with
+* `finished_code`: what you'll end up with after you complete the tutorial
+
+Note that this code doesn't include a server, so if you actually submit the form, you'll get an error in the Console. This exercise is left for the reader 😎
+
+## Setup
+
+If your computer is already running a local server, you don't need to set anything up at all! You can just type the relevant URL into your browser. Depending on where you've put this code and how you've set up your server, that URL will look something like this:
+
+`http://localhost/amp-script-tutorial/starter_code/index.html`
+
+Alternately, you can set up a quick local server using something like [serve](https://www.npmjs.com/package/serve), the Node.js-based static content server.
+
+Assuming you've installed [node](https://nodejs.org/), you can simply go to your command line, enter the directory where your code lives, and type:
```

Comment:
Please remove `simply` :)

Reply:
Ah, I thought you meant one or the other. You meant both!

There's probably a basic rule of writing here I'm not aware of. Perhaps, simply, remove unnecessary words? Or... should I say... "remove extra words"?

Reply:
Using words like "simply" and "just" assume these things will be easy for the reader when they may not be. Google's style guide does specifically point out using "simply": https://developers.google.com/style/tone?hl=pt-br

Reply:
I'm reading that... this is very interesting!

The word "just" is like a reflex for me. I've removed it, but as I re-edit it's so hard to avoid just sticking it back in. (see, there it is again!)

Reply:
Yeah, its sneaky! And it totally sounds like a good thing to be saying - weird huh?